tendrl/node_agent/message/handler.py

```diff
@@ -27,19 +27,18 @@ def read_socket(self, sock, address):
             self.data = sock.recv(RECEIVE_DATA_SIZE)
             message = Message.from_json(self.data)
             Logger(message)
-        except (socket_error, socket_timeout) as ex:
-            sys.stderr.write(
-                "Unable to read message from socket err.%s\n" % str(ex))
-        except (TypeError, ValueError, KeyError, AttributeError) as ex:
+        except (socket_error, socket_timeout):
+            traceback.print_exc(file=sys.stderr)
```

Comment:
Please use below way to print traceback

    exc_type, exc_value, exc_tb = sys.exc_info()
    traceback.print_exception(exc_type, exc_value, exc_tb, file=sys.stderr)


This is because traceback.print_exec is not greenlet safe